src/components/shop/shop.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Placeholder for shop screen.
+ *
+ * @module components/shop
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+import { Screen } from "../../core/screen.js";
+import { scrollableList } from "../../core/layout/scrollable-list/scrollable-list.js";
+import RexUIPlugin from "../../../lib/rexuiplugin.min.js";
+
+export class Shop extends Screen {
+    preload() {
```

Comment:
I'm a bit unsure about this format over using `this.plugins.installScenePlugin('rexUI', RexUIPlugin, "rexUI", this )`. It (supposedly) adds a second load call - which we should try and avoid/is uncesscary because it's been compiled in via an import). I honestly am unsure how this even can be working in a final build.

`installScenePlugin` (supposedly) just installs the plugin code.
Could see if it's loading it from `lib` on live.

Reply:
Both approaches work (I was failing on not passing `this` before), so happy to go the `installScenePlugin` route.

Reply:
33026bf